test/msgpack.cpp

```diff
@@ -220,5 +220,12 @@ TEST_CASE(test_msgpack_binary2)
     });
     EXPECT(migraphx::to_msgpack(bin) == msgpack_buffer(bin));
 }
+
+TEST_CASE(test_msgpack_binary_empty)
+{
+    migraphx::value::binary bin{};
+    EXPECT(migraphx::to_msgpack(bin) == msgpack_buffer(bin));
```

Comment:
You could just check the roundtrip would be the same.

Reply:
Done